### ISSUE TITLE
Staging Review finding: Accordion toggle buttons for refills aren't necessarily unique

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#101663]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** MHV Medications
**Product:** Prescription grouping
**Feature:** Linking similar meds

## Findings details
**VA.gov Experience Standard - issue:** [User can't identify an element or its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When viewing a medication details page, each accordion in the Refill History section has a toggle button with text "Refill - Filled on [date]." The date should be sufficient to make these buttons unique relative to one another.

However, when multiple prescriptions have "None noted" as their fill date, you end up with multiple accordion buttons with the same accessible name --- meaning some users (and especially screen reader users) are likely to have trouble distinguishing them from one another. Here's a [prescription for albuterol](https://staging.va.gov/my-health/medications/prescription/23844911) for test user KARSYN.GPSYSASYNCTWO where this is an issue.

<details><summary>Link, screenshot or steps to recreate</summary>

![Refill history section with four previous fills. Two refills have a fill date of "None noted."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/94684-Brian-1.png?raw=true)

</details>

## Recommended action

Not sure if this is a real issue or an issue with staging data only. But if there is a decent chance that you won't be able to display fill dates for all refills, you'll need to name each refill in some other unique way.

At midpoint, there was an index for each refill (Refill 1, Refill 2, Refill 3). Not sure if that got dropped in favor of the refill date, but I feel like you could do index plus date and be pretty certain that you'll always provide a unique accessible name.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [x] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by IdeaTrail** [comment]
This was a result of staging test data and not an isssue with the product, as discussed on the staging review.
